File: src/main/java/tfg/proyecto/TFG/dtos/DTOticketBajada.java
package tfg.proyecto.TFG.dtos;

import java.math.BigDecimal;
import java.time.LocalDateTime;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;
import tfg.proyecto.TFG.modelo.EstadoTicket;
@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class DTOticketBajada {
	private Long id;
    private String codigoQR;
    private LocalDateTime fechaCompra;
    private double precioPagado;
    private EstadoTicket estado;
    private Long usuarioId;
    private String nombreUsuario;
    
    private Long eventoId;
    private String nombreEvento;
}

File: frontend/appMovilShowPass/app/src/main/java/com/example/appmovilshowpass/data/remote/dto/DTOTicketBajada.kt
package com.example.appmovilshowpass.data.remote.dto

import com.example.appmovilshowpass.model.EstadoTicket

data class DTOTicketBajada (
    val id: Long,
    val codigoQR: String,
    val fechaCompra: String,
    val precioPagado: Double,
    val estado: EstadoTicket,
    val usuarioId: Long,
    val nombreUsuario: String,
    val eventoId: Long,
    val nombreEvento: String
)